### PR TITLE
feat: swipecontrols patch

### PR DESCRIPTION
**Pull request by TheJeterLP**

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledParentSignature.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.signature.implementation.method.MethodSignature
+import app.revanced.patcher.signature.implementation.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.signature.implementation.method.annotation.MatchingMethod
+import app.revanced.patches.youtube.layout.swipecontrols.annotations.SwipecontrolsCompatibility
+
+@Name("swipecontrolbrightnessenabled-parent-signature")
+@MatchingMethod(
+    "Ljvy;", "<init>"
```
Comment (Sculas) [inline src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledParentSignature.kt:12]: Have you updated this?

File: src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/annotations/SwipecontrolsCompatibility.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import app.revanced.patcher.annotation.Package
+
+@Compatibility(
+    [Package(
+        "com.google.android.youtube", arrayOf("17.24.34")
+    )]
+)
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+internal annotation class SwipecontrolsCompatibility
```
Comment (Sculas) [inline src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/annotations/SwipecontrolsCompatibility.kt:13]:
```suggestion
internal annotation class SwipeControlsCompatibility
```

File: src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledSignature.kt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.signature.implementation.method.MethodSignature
+import app.revanced.patcher.signature.implementation.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.signature.implementation.method.annotation.MatchingMethod
+import app.revanced.patches.youtube.layout.swipecontrols.annotations.SwipecontrolsCompatibility
+import org.jf.dexlib2.AccessFlags
+
+@Name("swipecontrolbrightnessenabled-signature")
+@MatchingMethod(
+    "Ljvy;", "<init>"
```
Comment (Sculas) [inline src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledSignature.kt:14]: Have you updated this?

**Review by Sculas — CHANGES_REQUESTED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledParentSignature.kt (hunk 1 above)
Comment (TheJeterLP) [reply src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledParentSignature.kt:12]: No, everything is pushed

Comment (Sculas) [reply src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledParentSignature.kt:12]: That's not what I meant. It looks like you copy-pasted this from another patch, without updating it accordingly. Could you do so, please?

Comment (TheJeterLP) [reply src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledParentSignature.kt:12]: Oh yes now I see what you mean. Will do tomorrow, I'm no longer at home

**Pull request closed without merging**